apps/x509.c

```diff
@@ -134,6 +135,9 @@ const OPTIONS x509_options[] = {
      "Serial number to use, overrides -CAserial"},
     {"next_serial", OPT_NEXT_SERIAL, '-',
      "Increment current certificate serial number"},
+    {"notbefore", OPT_NOTBEFORE, 's', "Cert notBefore, YYMMDDHHMMSSZ"},
+    {"notafter", OPT_NOTAFTER, 's',
+     "YYMMDDHHMMSSZ cert notAfter (overrides -days)"},
```

Comment:
Yeah, the naming can make some headaches. Especially as `-stardate` and `-enddate` already have different meaning in the `ca` and `x509` apps.
Does it make sense to also create aliases for the `ca` app, when using the new options for `req` and `x509` apps?

Reply:
> Yeah, the naming can make some headaches. Especially as `-startdate` and `-enddate` already have different meaning in the `ca` and `x509` apps. Does it make sense to also create aliases for the `ca` app, when using the new options for `req` and `x509` apps?

Yes, it would definitely make sense to add these new names as aliases in the ca app.

Reply:
@paulidale I have changed the option names to have underscores (`_`) now. Do you agree?
I will continue working on some tests then.

Reply:
I agree with the `_` in the names, making them better readable.
They are already in use for `x509` and other apps.

Reply:
Would be nice if you added, for consistency, `-not_before` and `-not_after` as aliases also to `ca.c`.